.github/workflows/release.yml

```diff
@@ -26,6 +26,8 @@ jobs:
                   bun install
                   bun run build
                   mkdir ${{ env.PLUGIN_NAME }}
+                  mkdir ${{ env.PLUGIN_NAME }}/themes
```

Comment:
Obsidian downloads the individual files `main.js`, `styles.css` and `manifest.json` from the release. It does not download the zip file.